emis-client/pom.xml

```diff
@@ -23,5 +25,31 @@
       <groupId>org.projectlombok</groupId>
       <artifactId>lombok</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.junit.jupiter</groupId>
+      <artifactId>junit-jupiter-api</artifactId>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.assertj</groupId>
+      <artifactId>assertj-core</artifactId>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>com.github.tomakehurst</groupId>
+      <artifactId>wiremock</artifactId>
+      <version>${wiremock.version}</version>
+      <scope>test</scope>
+    </dependency>
   </dependencies>
+  <build>
+    <plugins>
+      <!-- If this plugin is not included, then maven will literally not even run the unit tests. -->
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-surefire-plugin</artifactId>
```

Comment:
this doesn't smell right .. health-apis-parent includes surefire.

Reply:
For some reason it wasn't until I added this that the unit tests ran with a `mvn clean install`. This repo originally never even had any unit tests.

Reply:
I found out that for some reason, surefire plugin version `2.22.2` doesn't run the tests, but version `3.0.0-M7` ends up running the tests. It's still a bad smell though because MPI is able to go and run the tests just fine with version `2.22.2`. I have no idea why. Also.. I _don't_ need to specify junit jupiter as a dependency in MPI, but I _do_ here in emis. There is something very weird going on & I don't know what.